packages/embedded/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fiberplane/embedded",
```

Comment:
also maybe rename this to just `fiberplane`?

Reply:
per conversation, changing to `@fiberplane/hono`